### PR TITLE
get user profile tooltip to API call

### PR DESCRIPTION
**Pull request by alkeshrails**
Make the tooltips of users come via API instead of pre-rendered!
Monday : https://aicrowd.monday.com/boards/363093650/pulses/522768696

User tooltip
Screenshot from 2020-04-24 13-25-05

### CODE REVIEW

File: app/javascript/stimulus_js/controllers/participants_controller.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Controller } from 'stimulus';
+export default class extends Controller {
+
+  getUserProfile(event){
+    const avatar        = this.data.get('avatar');
+    const location      = this.data.get('location');
+    const username      = this.data.get('username');
+    const participantId = this.data.get('participant');
+
+    $.ajax({
```
Comment (piotrekpasciak) [inline app/javascript/stimulus_js/controllers/participants_controller.js:10]: We should avoid jQuery whenever possible as it's legacy library. Please use vanilla javascript method e.g `fetch` or `async await`:
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch
https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await

File: app/views/participants/_user_link.html.erb [hunk 2]
```diff
@@ -1,17 +1,14 @@
-<% 
-    uniq_id = rand(1000)
-    random_class = 'tooltip-user-wrapper-' + uniq_id.to_s + '-' + participant.id.to_s
+<% avatar = avatar.present? ? avatar : true %>
+<% username = username.present? ? username : true %>
+<%
+  uniq_id = rand(1000)
```
Comment (piotrekpasciak) [inline app/views/participants/_user_link.html.erb:4]: SecureRandom.hex()

File: app/views/participants/_name_avatar.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<% if !defined?(avatar) || (defined?(avatar) && avatar && avatar != 'false') %>
```
Comment (piotrekpasciak) [inline app/views/participants/_name_avatar.html.erb:1]: `(defined?(avatar) && avatar != 'false')`  will be simpler and work the same way.

File: app/views/participants/_name_avatar.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+  <%= participant_avatar participant %>
+<% end %>
+<% if !defined?(username) || (defined?(username) && username && username != 'false') %>
```
Comment (piotrekpasciak) [inline app/views/participants/_name_avatar.html.erb:4]: (defined?(username) && username != 'false')

File: app/javascript/stimulus_js/controllers/participants_controller.js (hunk 1 above)
Comment (alkeshrails) [reply app/javascript/stimulus_js/controllers/participants_controller.js:10]: By using `fetch` I didn't get `render partial` of tooltip in response, that we need to set as `html` .

**Pull request merged**